src/Pact/Analyze/Types/Eval.hs

```diff
@@ -140,6 +140,9 @@ newtype Constraints
 instance Show Constraints where
   show _ = "<symbolic>"
 
+instance Semigroup Constraints where
```

Comment:
Does Monoid require Semigroup instances now?

Reply:
Yes, `Semigroup` is a superclass of `Monoid` now.